curated/ORFs/uniprot_sars-cov-2.gpi

```diff
@@ -43,14 +43,14 @@ UniProtKB	P0DTC4	E protein	Envelope small membrane protein	E|4|E protein	protein
 UniProtKB	P0DTC4-PRO_0000449651	E protein	Envelope small membrane protein	E protein|sM protein|P0DTC4(1-75)|E|4	protein	taxon:2697049	UniProtKB:P0DTC4
 UniProtKB	P0DTC5	P0DTC5	Membrane protein		protein	taxon:2697049	
 UniProtKB	P0DTC5-PRO_0000449652	M protein	Membrane protein	M protein|P0DTC5(1-222)	protein	taxon:2697049	UniProtKB:P0DTC5
-UniProtKB	P0DTC6	ORF6	Non-structural protein 6	ORF6	protein	taxon:2697049	
-UniProtKB	P0DTC6-PRO_0000449653	ns6	ORF6 protein	ns6|P0DTC6(1-61)|6	protein	taxon:2697049	UniProtKB:P0DTC6
+UniProtKB	P0DTC6	ORF6	Non-structural protein 6	ORF6|ns6|6	protein	taxon:2697049	
```

Comment:
This is what I have in master:

`UniProtKB	P0DTC6	ORF6	ORF6	ORF6|accessory protein 6|ns6|X3	protein	taxon:2697049	`

I think this is correct. The nsps shouldn't be confused with the ns series